app/views/school_groups/_sub_nav.html.erb

```diff
@@ -21,6 +21,7 @@
             <div class="dropdown-menu scrollable" aria-labelledby="manage-school-group" id="dropdown-manage-school-group">
               <%= link_to t('school_groups.sub_nav.chart_settings'), school_group_chart_updates_path(@school_group), class: 'dropdown-item' %>
               <%= link_to t('school_groups.sub_nav.clusters'), school_group_clusters_path(@school_group), class: 'dropdown-item' %>
+              <%= link_to t('manage_school_menu.manage_tariffs'), school_group_energy_tariffs_path(@school_group), class: 'dropdown-item' %>
```

Comment:
"We won’t make this editor available to group admins initially, so links to the new pages should be in the ES admin only section of the Manage Group menu". Can you move this into the admin section?

If this impacts the tests, then you could comment those out for now. I want to allow us to internally test this out before making it visible to users.

Reply:
done :+1: (see skipped test)